### ISSUE TITLE
Unknown function: $error

### ISSUE THREAD

**Issue by sajee1** [IBM/JSONata4Java#123]
Hi I am trying to produce custom error using $error function, but it returns Unknown function: $error.
Expressions expr = Expressions.parse("$error('Custom Error')");
JsonNode result = expr.evaluate(jsonObj);

I am using version 1.4.6 (the latest in Maven repo, 1.4.7 is not present in maven central Repo https://mvnrepository.com/artifact/com.ibm.jsonata4java/JSONata4Java)

**Comment by wnm3** [comment]
I have added the $error function. It will throw a EvaluateRuntimeException using the string provided as its only argument.  Test in BasicExpressionTest:
```
      // test $error call
      try {
         expression = Expressions.parse("$error(\"Custom Exception\")");
         JsonNode result = expression.evaluate(null);
         assertTrue("Should not get here - exception should be thrown", false);
      } catch (EvaluateException e) {
         String msg = e.getMessage();
         assertTrue("was " + msg, msg.equals("Custom Exception"));
      }
```

**Comment by wnm3** [comment]
If this problem is resolved in version 1.4.8 to be released soon, please close this issue.
